src/widgets/ResultRow.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2016
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, TouchableOpacity, StyleSheet } from 'react-native';
+import Icon from 'react-native-vector-icons/Ionicons';
+import { APP_FONT_FAMILY, SUSSOL_ORANGE, BACKGROUND_COLOR, GREY } from '../globalStyles';
+
+const ResultRow = props => {
+  const { data, renderLeftText, renderRightText, onPress, isSelected, showCheckIcon } = props;
+  const selected = isSelected(data);
```

Comment:
bit confusing - the `isSelected` passed doesn't take a parameter. Also, this should just be a boolean

Reply:
https://github.com/openmsupply/mobile/blob/2bbd3568e37522d256bea82bdc02ba1259f466f5/src/widgets/MultiSelectList.js#L135

@joshxg It is declared here and passed in as prop to ResultRow.
Also it is a boolean,